partner_contact_in_several_companies/tests/test_partner_contact_in_several_companies.py

```diff
@@ -158,45 +153,41 @@ def test_03_search_match_attached_contacts(self):
                                              'set_value': False
                                              }}
         partner_ids = self.partner.search(
-            cr, uid,
-            [('parent_id', 'ilike', 'YourCompany')],
-            context=ctx
-        )
+            [('parent_id', 'ilike', 'YourCompany')])
         self.assertIn(self.bob_contact_id, partner_ids, )
 
     def test_04_contact_creation(self):
         """Check that we're begin to create a contact"""
-        cr, uid = self.cr, self.uid
 
         # Create a contact using only name
-        new_contact_id = self.partner.create(cr, uid, {'name': 'Bob Egnops'})
+        new_contact_id = self.partner.create({'name': 'Bob Egnops'})
         self.assertEqual(
             self.partner.browse(cr, uid, new_contact_id).contact_type,
```

Comment:
`new_contact_id` is a recordset, remove the browse method call:
`self.assertEqual(new_contact_id.contact_type, 'standalone')`